test/Conversion/MooreToCore/basic.mlir

```diff
@@ -836,3 +836,16 @@ dbg.variable "b", %dbg0 scope %dbg1 : !moore.l32
 dbg.array [%dbg0] : !moore.l32
 // CHECK: dbg.struct {"q": [[TMP]]} : i32
 dbg.struct {"q": %dbg0} : !moore.l32
+
+// CHECK-LABEL: hw.module @Assert
+moore.module @Assert(in %cond : !moore.l1)  {
+  moore.procedure always {
+  // CHECK: verif.assert %cond label "" : i1
+  moore.assert immediate %cond : l1
+  // CHECK: verif.assume %cond label "" : i1
+  moore.assume observed %cond  : l1
+  // CHECK: verif.cover %cond label "" : i1
+  moore.cover final %cond : l1
```

Comment:
Can you also add a check that has a non-empty label?

Reply:
Sure, sorry for missing this check